.appveyor.yml

```diff
@@ -71,37 +71,37 @@ for:
       - image: Visual Studio 2013
   install:
   - cmd: >-
-      appveyor DownloadFile https://github.com/appleseedhq/appleseed-deps/releases/download/v17/appleseed-deps-stage-win64-vs120-v17.zip
+      appveyor DownloadFile https://github.com/appleseedhq/appleseed-deps/releases/download/v17/appleseed-deps-stage-win64-vs110-v17.zip
 
-      appveyor DownloadFile https://github.com/appleseedhq/appveyor-deps/releases/download/v1/boost_1_55_0_2013_py35.zip
+      appveyor DownloadFile https://www.dropbox.com/s/fwjzqc2m1a5euej/boost_1_55_0_vc11.zip?dl=1
```

Comment:
You shouldn't rely on that file being in Dropbox, it's shared from my account. You should upload it to GitHub like you did with `boost_1_55_0_2013_py35.zip`.